### ISSUE TITLE
Published dmg tried to hit yahoo keykeeper

### ISSUE THREAD

**Issue by mikekap** [jschauma/jass#15]
Using jass from the installer in the readme results in a (very) long pause before succeeding, but with this message:
```
$ echo "hi" | jass -G -u torvalds -v
=> Checking that all variables look ok...
=> Encrypting...
Unable to GET https://keykeeper.corp.yahoo.com/v2/api/keys?user=torvalds: Get https://keykeeper.corp.yahoo.com/v2/api/keys?user=torvalds: dial tcp 98.136.205.33:443: i/o timeout
=> Converting pubkeys to PKCS8 format...
begin-base64 600 message
U2FsdGVkX19+DU+Hibstv19qPs8s5D0OXfM3D6Iu70A=
====
begin-base64 600 torvalds-aa:b0:9f:c2:f8:45:57:1b:22:a9:d4:50:49:bb:8b:58
BvjvVYZhnom2iyqzjo2OR8xQZJY9Dooineen/dwPgmktLRIbK7L/JATVYDe4LSERrh/Cvo/ifMtz
zAVktUls3CB6FChWJWBR4Y8OJDIXKo11ED8/0u4L07x1cUVY0jMAqeg+uDcV47IrJOIR1ZUkkz1/
AVldXDh1Hj+iSEXqsxvzUQ1ks7Fk9efLNuImCEzDzBKRstdehLvbj9QmAMGqUvRga7LWxgXNJgsJ
q9rabEnmMTE4VD1Cbwk+4fcGAa1OgvDnsCued3EJw/qHTIirJx2cg8aaFbXIv3AhhNnqWLmnuanz
IrPfzUtlY6eeawFvsY0/ZFR5ZVXLgayvZse6KQ==
====
begin-base64 600 version
VkVSU0lPTjogNC4xCg==
====
```

The installer likely needs some repackaging.

**Comment by jschauma** [comment]
Ah, thanks for letting me know.  Looks like I packaged the Yahoo version, not the public version. (The only difference is that for Yahoo, keykeeper is enabled by default, as you saw.)

I've rebuilt the DMG and updated it.  (I haven't bumped the version, since the version references the jass version itself, not the dmg.)  If you fetch https://www.netmeister.org/apps/jass-4.1.dmg (signature: https://www.netmeister.org/apps/jass-4.1.dmg.asc), that should contain the binary without the internal URL.
